src/main/java/seedu/duke/entities/Order.java

```diff
@@ -32,4 +32,12 @@ public double getTotalPrice() {
         return totalPrice;
     }
 
+    public void printOrder() {
```

Comment:
actually, you can override the `toString` method to return a string representation of the order, no need to create a function to print it.

Reply:
OK